File: src/market-tracker-webapi/Application/Domain/ShoppingList.cs
namespace market_tracker_webapi.Application.Domain;

public class ShoppingList
{
    public int Id { get; set; }
    
    public required string Name { get; set; }
    
    public DateTime? ArchivedAt { get; set; }
    
    public DateTime CreatedAt { get; set; }
    
    public required Guid OwnerId { get; set; }
}

File: src/market-tracker-android/app/src/main/java/pt/isel/markettracker/domain/model/list/ShoppingList.kt
package pt.isel.markettracker.domain.model.list

import java.time.LocalDateTime

data class ShoppingList(
    val id: String,
    val name: String,
    val archivedAt: LocalDateTime?,
    val createdAt: LocalDateTime,
    val ownerId: String,
    val isOwner: Boolean,
    val isArchived: Boolean,
    val numberOfMembers: Int,
)